static/select_seats.css

```diff
@@ -10,25 +10,38 @@ div.seat-info div {
 
 .theater-seat-map svg {
     max-width: 400px;
-    max-height: 400px;
+    max-height: 480px;
+    viewBox: "0 0 400 400"
```

Comment:
Yeah, I think this is leftovers from when I tried. It did not work. I just forgot to remove it. I'll do that before merging.